### PR TITLE
More comments

### PR DESCRIPTION
**Pull request by Miepee**

### CODE REVIEW

File: Underanalyzer/Mock/GameContextMock.cs [hunk 1]
```diff
@@ -28,24 +55,31 @@ public void DefineMockAsset(int assetIndex, string assetName, AssetType assetTyp
...
         assets[assetIndex] = assetName;
     }
-    public string GetMockAsset(int assetIndex, AssetType assetType)
+    
+    /// <summary>
+    /// Fetches an asset from <see cref="MockAssets"/>.
+    /// </summary>
+    /// <param name="assetType">The asset type of the asset that should be fetched.</param>
+    /// <param name="assetIndex">The index of the asset that should be fetched.</param>
+    /// <returns></returns>
```
Comment (colinator27) [inline Underanalyzer/Mock/GameContextMock.cs:64]: Probably should either not have returns, or just say that it returns the string name of the asset, or null if not found.

File: Underanalyzer/Mock/GameContextMock.cs [hunk 2]
```diff
@@ -28,24 +55,31 @@ public void DefineMockAsset(int assetIndex, string assetName, AssetType assetTyp
...
-
+    
+    /// <inheritdoc/>
     public string GetAssetName(int assetIndex, AssetType assetType)
     {
         return assetType switch
         {
             AssetType.RoomInstance => assetIndex >= 100000 ? $"inst_id_{assetIndex}" : null,
-            _ => GetMockAsset(assetIndex, assetType)
+            _ => GetMockAsset(assetType, assetIndex)
```
Comment (colinator27) [inline Underanalyzer/Mock/GameContextMock.cs:82]: We probably also want to change the `GetAssetName` parameter order as well. This was originally done in the `index, type` order for consistency.

File: Underanalyzer/Mock/GameContextMock.cs (hunk 1 above)
Comment (Miepee) [reply Underanalyzer/Mock/GameContextMock.cs:64]: missed this

File: Underanalyzer/Mock/GameContextMock.cs (hunk 2 above)
Comment (Miepee) [reply Underanalyzer/Mock/GameContextMock.cs:82]: missed this too.

**Pull request merged**